drivers/wifi/nrf700x/Kconfig

```diff
@@ -152,6 +152,14 @@ config NRF_WIFI_FW_PATCH_INTEGRITY_CHECK
 	  SHA-256 verification algorithm.This option impacts the loading time of the
 	  nRF70 FW patches but protects against corrupted FW patches.
 
+config NRF_WIFI_FW_PATCH_DFU
+	bool "Direct Firmware Update of nrf70 FW patch"
+	depends on PARTITION_MANAGER_ENABLED
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enables DFU support for nRF70 FW patch. This option requires an additional mcuboot partition pair
+	  defined in the pm_static.yaml file.
```

Comment:
may be add a sentence about requirement of merging and signing images, as that is the only impact in the driver?